guides/common/modules/proc_creating-custom-snippets-to-apply-errata.adoc

```diff
@@ -0,0 +1,35 @@
+[id="creating-custom-snippets-to-apply-errata_{context}"]
+= Creating custom snippets to apply errata
+
+You can use custom snippets to execute code before and/or after applying errata on hosts.
+
+.Prerequisites
+* Depending on the template, multiple custom snippet hooks exist which you can use to include custom snippets.
+Check your template to verify which custom snippets you can use.
++
+You can view used job templates under *Administer* > *Remote Execution Features*.
```

Comment:
I don't think "used job templates" is a great way to phrase it. It sounds like 'used underwear'. 

Would "previously applied job templates" work?

Reply:
:joy: 

By "used", I mean job templates that are executed when you click on specific REX job buttons in the WebUI. -> Reworded.